feat(node): Update integrations pages (#9734)
getsentry/sentry-docs: docs/platforms/javascript/guides/aws-lambda/configuration/integrations/contextlines.mdx

---
title: ContextLines
description: "Adds source code from inline JavaScript of the Node JavaScript files. (default)"
---

_Import name: `Sentry.contextLinesIntegration`_

> This integration adds source code from inline JavaScript of the Node JavaScript files.

Reviewer: I'm having trouble parsing what's meant by "inline JavaScript of the Node JavaScript files"

Do you mean 
```suggestion
This integration adds source code from inline JavaScript used in Node JavaScript files.
```
or maybe
```suggestion
This integration adds source code from inline JavaScript to the Node JavaScript files.
```

or something else?

Author: sorry, this is a bit confusing - I updated it, LMK if it is clearer now...?

Reviewer: Yes, much clearer now, thanks!

---
title: ContextLines

> description: "Adds source code to your stack frames. (default)"

Author: @stephanie-anderson the reason I made these separate pages instead of solving this via `<PlatformSection>` in the common file is that the description has to be different - it is default for node, but not default for browser 😬 or is it possible to have a conditional description for certain guides? 🤔
